VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/context-menu-header/context-menu-header.component.scss

```
@@ -0,0 +1,6 @@
+@import '~govuk-frontend/govuk/base';
+
+.horizontal-spacer {
+  padding-bottom: 10px;
+  border-bottom: 2px solid govuk-colour('mid-grey');
+}
```

If this is used multiple places we should place this in a common file.
```suggestion
@import 'abstracts';

.horizontal-spacer {
  padding-bottom: rem(10px);
  border-bottom: 2px solid govuk-colour('mid-grey');
}
```

---

I've seen this used elsewhere if so we should move this to a common file.

---

Are we sure about this as I copied over existing styles or do I need to check with Neil?

---

You need to include @import 'abstracts'; at the top of the file.